ox3apiclient/__init__.py

```diff
@@ -42,13 +39,20 @@
 API_PATH_V2 = '/ox/4.0'
 API_PATH_SSO = '/api'
 ACCEPTABLE_PATHS = (API_PATH_V1, API_PATH_V2, API_PATH_SSO)
-JSON_PATHS = (API_PATH_V2,)
+JSON_PATHS = (API_PATH_V2)
```

Comment:
We still need the comma here, otherwise it won't be a tuple. But we can still merge it and I'll fix it in my upcoming PR.